### PR TITLE
Return response object instead of string ID

### PR DESCRIPTION
**Pull request by Dimedrolity**
Now caller (at least [Ansible-role](https://github.com/moira-alert/moira-trigger-role/pull/24)) can show source error text.

### CODE REVIEW

File: tests/models/test_trigger.py [hunk 1]
```diff
@@ -142,7 +142,7 @@ def test_save_new_trigger(self):
                 self.assertTrue(get_mock.called)
                 self.assertTrue(put_mock.called)
                 self.assertEqual(put_mock.call_args[0][0], f'trigger?{self.QUERY_PARAM_VALIDATE_FLAG}')
-                self.assertEqual(result_id, trigger_id)
+                self.assertEqual(result_id['id'], trigger_id)
```
Comment (Tetrergeru) [inline tests/models/test_trigger.py]:
```suggestion
                self.assertEqual(result['id'], trigger_id)
```

File: tests/models/test_trigger.py [hunk 2]
```diff
@@ -172,7 +172,7 @@ def test_save_existing_trigger(self):
                 self.assertTrue(get_mock.called)
                 self.assertTrue(put_mock.called)
                 self.assertEqual(put_mock.call_args[0][0], f'trigger/{trigger_id}?{self.QUERY_PARAM_VALIDATE_FLAG}')
-                self.assertEqual(result_id, trigger_id)
+                self.assertEqual(result_id['id'], trigger_id)
```
Comment (Tetrergeru) [inline tests/models/test_trigger.py]:
```suggestion
                self.assertEqual(result['id'], trigger_id)
```

File: tests/models/test_trigger.py [hunk 3]
```diff
@@ -198,4 +198,4 @@ def test_save_trigger_with_id(self):
...
```
Comment (Tetrergeru) [inline tests/models/test_trigger.py]:
```suggestion
                self.assertEqual(result['id'], trigger_id)
```

**Review by Tetrergeru — DISMISSED** [review]

File: tests/models/test_trigger.py (hunk 1 above)
Comment (Dimedrolity) [reply tests/models/test_trigger.py]: Thx, fixed 3 times https://github.com/moira-alert/python-moira-client/pull/27/commits/61ea6d2f24fc7bf235c1f19de725bbf14950fd98

**Review by Tetrergeru — APPROVED** [review]

**Review by kissken — APPROVED** [review]

**Pull request merged**